README.md

```diff
@@ -1,22 +1,29 @@
+# Kubeapps
+
 [![Build Status](https://travis-ci.org/kubeapps/kubeapps.svg?branch=master)](https://travis-ci.org/kubeapps/kubeapps)
 
-# Kubeapps Installer
+<img src="https://github.com/kubeapps/kubeapps/raw/master/img/logo.png" width="100">
```

Comment:
the link doesn't work

Reply:
Hopefully it will work as soon as it is merged :)